public/plugins/ckeditor/plugins/videodetector/dialogs/videoDialog.js

```diff
@@ -0,0 +1,5 @@
+﻿CKEDITOR.dialog.add("videoDialog",function(a){return{title:"Insert a Youtube, Vimeo, Dailymotion URL or embed code",minWidth:400,minHeight:100,contents:[{id:"tab-basic",label:"Basic Settings",elements:[{type:"text",id:"url_video",label:"Youtube, Vimeo, Dailymotion URL or embed code",validate:CKEDITOR.dialog.validate.notEmpty("Empty!")}]}],onOk:function(){var b=detectar(),c="";"youtube"==b.reproductor?c="https://www.youtube.com/embed/"+b.id_video+"?autohide=1&controls=1&showinfo=0":"vimeo"==b.reproductor?
+c="https://player.vimeo.com/video/"+b.id_video+"?portrait=0":"dailymotion"==b.reproductor&&(c="https://www.dailymotion.com/embed/video/"+b.id_video);b=new CKEDITOR.dom.element("div");b.setAttribute("class","videodetector");var d=new CKEDITOR.dom.element("iframe");d.setAttribute("src",c);d.setAttribute("frameborder","0");b.append(d);a.insertElement(b)}}});
+function detectar(){var a=document.getElementsByClassName("cke_dialog_contents").item(0).getElementsByTagName("input").item(0).value,b="",c="";0<=a.indexOf("youtu.be")&&(c="youtube",b=a.substring(a.lastIndexOf("/")+1,a.length));0<=a.indexOf("youtube")&&(c="youtube",0<=a.indexOf("</iframe>")?(b=a.substring(a.indexOf("embed/")+6,a.length),b=b.substring(b.indexOf('"'),0)):b=0<=a.indexOf("&")?a.substring(a.indexOf("?v=")+3,a.indexOf("&")):a.substring(a.indexOf("?v=")+3,a.length));0<=a.indexOf("vimeo")&&
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.<br>Mixed double and single quotes.